forte/data/readers/audio_reader.py

```diff
@@ -53,21 +54,27 @@ def _collect(self, audio_directory) -> Iterator[Any]:  # type: ignore
 
         Returns: Iterator over paths to audio files
         """
-        return dataset_path_iterator(audio_directory, self.configs.file_ext)
+        # construct ImageMeta and store it in DataPack
+        return dataset_path_iterator(
+            audio_directory,
+            self.configs.file_ext,
+        )
 
     def _cache_key_function(self, audio_file: str) -> str:
         return os.path.basename(audio_file)
 
     def _parse_pack(self, file_path: str) -> Iterator[DataPack]:
         pack: DataPack = DataPack()
-
+        payload_idx = 0
```

Comment:
I think we don't need to change the `AudioReader` in this PR since the design of lazy loading is not complete. You can try to address this issue in the PR to #841